src/js/extractors/factories/callExpression.ts

```diff
@@ -20,6 +21,24 @@ export function callExpressionExtractor(calleeName: string | string[], options:
 
     validateOptions(options);
 
+    let contentOptions: IContentOptions = {
+        trimWhiteSpace: true,
+        preserveIndentation: false,
```

Comment:
I'm not sure the defaults for `trimWhiteSpace` and `preserveIndentation` make sense like this... Normally I would expect the extracted string to match the value at runtime, will not be trimmed and might contain indentation whitespace.

Reply:
It's okay for me, to not manipulated extracted strings by default. I simply took the same defaults as for the element content extraction.

Reply:
I thought so. The difference between this and the element content is that in HTML whitespace follows different rules than inside of a JS string. Besides, having these defaults would be a breaking change as `'    foo     '` would suddenly get extracted as `'foo'`.
Please change it to `trimWhiteSpace: false` and `preserveIndentation: true`

Reply:
Done.